src/tasks/components/TaskHeaderCard.tsx

```diff
@@ -0,0 +1,175 @@
+// Libraries
+import React, {PureComponent} from 'react'
+import {connect, ConnectedProps} from 'react-redux'
+import {withRouter, RouteComponentProps} from 'react-router-dom'
+import Moment from 'react-moment';
+
+// Components
+import {
+  SlideToggle,
+  ComponentSize,
+  ResourceCard,
+  InputLabel,
+  FlexBox,
+  AlignItems,
+  FlexDirection,
+  JustifyContent,
+} from '@influxdata/clockface'
+
+// Actions
+import {addTaskLabel, deleteTaskLabel, runTask, getRuns} from 'src/tasks/actions/thunks'
+
+// Types
+import {ComponentColor, Button} from '@influxdata/clockface'
+import {Task, AppState} from 'src/types'
+
+interface PassedProps {
+  task: Task
+  onActivate: (task: Task) => void
+  onRunTask: (taskID: string) => void
+}
+
+type ReduxProps = ConnectedProps<typeof connector>
+type Props = PassedProps & ReduxProps
+
+export class TaskHeaderCard extends PureComponent<
```

Comment:
same as the previous point about PureComponent vs Functional Component, although I could be wrong